pkg/networkmanager/utils.js

```diff
@@ -85,7 +85,7 @@ export function ip4_to_text(num, zero_is_empty) {
 }
 
 export function ip4_from_text(text, empty_is_zero) {
-    function invalid() {
+    function invalid(id) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19024-20230626-161008-14729dcc-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)